src/Illuminate/Routing/Router.php

```diff
@@ -1089,6 +1089,26 @@ public function currentRouteUses($action)
     }
 
     /**
+     * Determine if the current action matches a given action. This take
```

Comment:
Thanks, but what do you mean by "cs"? coding style? I can fix it but I need some clue about it